x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/product_selector/product_selector.tsx

```diff
@@ -205,7 +205,7 @@ export const ProductSelector: React.FC<ProductSelectorProps> = ({
                       defaultMessage: 'Search UI for App Search',
                     }
                   ),
-                  to: '', // TODO Update docLink
+                  to: 'https://docs.elastic.co/search-ui/tutorials/app-search',
```

Comment:
>                  to: 'https://docs.elastic.co/search-ui/tutorials/app-search',

If you want to add these to the doc link service too, I think that should be feasible